e2e/components/Details.test.ts

```diff
@@ -0,0 +1,43 @@
+import {test, expect} from '@playwright/test'
+import {visit} from '../test-helpers/storybook'
+import {themes} from '../test-helpers/themes'
+
+test.describe('Details', () => {
+  test.describe('Default', () => {
+    for (const theme of themes) {
+      test.describe(theme, () => {
+        test('default @vrt', async ({page}) => {
+          await visit(page, {
+            id: 'components-details--default',
+            globals: {
+              colorScheme: theme,
+            },
+          })
+
+          // Default state - closed
+          expect(await page.screenshot()).toMatchSnapshot(`Details.Default.${theme}.png`)
+          // Click the summary to open
+          await page.getByText('See Details').click()
+          // Open state
+          expect(await page.screenshot()).toMatchSnapshot(`Details.Default.${theme}.png`)
```

Comment:
```suggestion
          // Default state - closed
          expect(await page.screenshot()).toMatchSnapshot(`Details.Default.${theme}.png`)
          // Click the summary to open
          await page.getByText('See Details').click()
          await page.getByText('This is some content').waitFor();
          // Open state
          expect(await page.screenshot()).toMatchSnapshot(`Details.Default.${theme}.open.png`)
```

Reply:
Thank you so much! I applied it on my local - sometime prettier makes trouble if I push directly here